dtos/http_request.go

```diff
@@ -0,0 +1,28 @@
+package dtos
+
+type FormCreationRequest struct {
+	Title  string `json:"title"`
+	UserId int64  `json:"userId"`
+}
+
+type FormUpdateRequest struct {
+	Title                 string `json:"title"`
+	SendEmailOnSubmission bool   `json:"sendEmailOnSubmission"`
+	UserId                int64  `json:"userId"`
+}
+
+// use for creating/updating question
+type QuestionUpdateRequest struct {
```

Comment:
- `UpdateQuestionRequestDto` 
- will we also not need id of the question to be updated?